### PR TITLE
Change directories to the project root in each script

### PR DESCRIPTION
**Pull request by jbarnette**
@maddox This isn't foolproof, but a lot of code assumes things are executed from the project root.

### CODE REVIEW

**Comment by maddox** [comment]
Should it be at the end of each script instead, to clean things up after each execution?

Or, at least, after the execution of the other script calls (at the top), to ensure they didn't leave us in another directory. (the first option would always ensure this though.)

File: script/bootstrap [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 set -e
 
+cd $(dirname "$0")/..
```
Comment (tmm1) [inline script/bootstrap]: you'll want quotes around this so it handles directory names with spaces

**Comment by jbarnette** [comment]
@maddox Subscripts execute in subprocesses, so they won't change the parent process's working directory.

File: script/bootstrap (hunk 1 above)
Comment (jbarnette) [inline script/bootstrap]: :+1: I'll double all these up.

**Comment by jbarnette** [comment]
@maddox CLA was smooth :zap:

**Comment by maddox** [comment]
> Subscripts execute in subprocesses, so they won't change the parent process's working directory.

lol :blush:

**Comment by jbarnette** [comment]
:bow: @maddox I'll push you up another PR with some extra quote safety for the `cd`.

**Pull request merged**